src/main/java/com/google/sps/servlets/startGameServlet.java

```diff
@@ -38,7 +39,20 @@ public startGameServlet(){
  
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
       String gameName = request.getParameter("game-name");
+      String quizQuestion = "hardCodedTemp";
+      long quizTime = 0;
+
+      // TODO: check if game name is valid (needs user entity and GameUtils merged)
+      // Query query = new Query("User");
+      // PreparedQuery results = datastore.prepare(query);
+      // List<Entity> resultsList = results.asList();
+      // GameUtils gameUtils = new GameUtils();
+      // if(!gameUtils.isValidGameName(gameName, resultsList, givenUser)){
```

Comment:
is it possible to query for a specific user name rather than return the entire list of user entities?

Reply:
I looked into it a little, it looks like you can use a filter like this:      
Filter getCorrectUser = new FilterPredicate("userID", FilterOperator.EQUAL, givenUser);
Query query = new Query("Game").setFilter(getCorrectUser);
PreparedQuery results = datastore.prepare(query);
I tried it out with my game entity and the game name property, and it returns the correct entity. I can't test if it will work with the user entity, but I'm assuming it will return the correct entity. I added that to the commented section